google-platforms-case-study.md

```diff
@@ -0,0 +1,185 @@
+  (Analytics Help, 2023)
+
+These reports will provide great insights into how people found your website and/or what made them visit.
+examples of channels or sources people could have arrived from and triggered them to visit could be:
+
+- Paid Social ads, like ads on facebook or instagram.
+- Paid search, like paid ads on google.
+- Organic search (no paid), like in the search result of google.
+- Direct traffic, people who just enter your website in the browser and go there.
+
+#### Engagement reports
+
+In GA4 you will find four different types of engagement reports
+
+- Events, could be anything happening or user action on the page. Like: page view, button click, form submit, scrolled.
+- Conversions, is an event, but usually an event which is view as the final step in a number of events. Like: making a purchase or complete registration.
+- Pages and screens, lets you view on which page the engagement is happening. How many views a certain page has, total events triggered on a certain page or how many users visited the certain page.
+- Landing pages, gives you an overview of the engagement on the different landing pages (the first page of the website people viewed when entering the website)
+
+#### Demographics reports
+
+Lets you see from what city or country your traffic/users are coming from.
+
+#### Exploration reports
+
+Makes you customize your own reports, which could provide greater insights to different user journeys.
+
+In GA4 you could setup a lott of complex reports, which might fit your needs better insights then whats provided by the standard reports.
+Lets say you wanted to view a report with the following conditions: Users who made a purchase, but only users who clicked the primary button on the home page and which did not arrive from paid ads. You could add all these filters each time on one of the standard reports, but if this is a report you would like to view more frequently, to make an exploration report for it would be much more efficient.
+
+Here you could also create more visual charts and tables, like: funnel exploration and path exploration, to get a visualization on the user journey and behavior.
+
+#### Other features
+
+A list of other features provided by GA4:
+
+- Enhanced Measurement, lets you track page views for SPAs (Single page apps).
+- Audiences, GA4 let you create different audiences, which you can customize yourself. i.e. Only users who made a purchase could be a audience.
+- Traffic filters, Filter traffic from certain ip-addresses. i.e. Internal traffic.
+- Unwanted referrals filter, Filter traffic from websites you don't want to show up in your reports.
+- Sessions, let you define what should be counted as a session(a user interacting with your website).
+- Engaged session, let you define how long time
+-
+
+### Google Tag Manager
+
+As mentioned in the introduction, I will very briefly cover the platform google tag manager(GTM) and how it works, and how it works in relation to google analytics.
```

Comment:
There is no **I** in technical literature (I mean, there are two, but...).

We use `the author` to refer to ourselves instead of I. In this case I think you can shorten this to avoid it completely.